audio_filters/Audio_visualizer_DSP.py

```diff
@@ -0,0 +1,312 @@
+"""python application to view audio equalizer*
+of input sound source"""
+import struct
+import sys
+
+import numpy as np
+import pyaudio
+import pyqtgraph as pg
+from pynput.keyboard import Key, Listener
+from pyqtgraph.Qt import QtCore, QtGui
+from scipy.fftpack import fft
+
+
+class AudioStream:
+    """stream audio from input source (mic) and continuously
+    plot (bar) based on audio spectrum from waveform data"""
+
+    def __init__(self, num, symbol):
```

Comment:
It is a bot therefore it is persistent.  If you say that this is resolved but it is not then the bot will keep saying it is not.
```suggestion
    def __init__(self, num: int, symbol: int) -> None:
```